server/neptune/workflows/internal/job/root.go

```diff
@@ -0,0 +1,32 @@
+package job
+
+import (
+	"path/filepath"
+
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/github"
+)
+
+type Root struct {
+	Name      string
+	Path      string
+	TfVersion string
```

Comment:
delete?

Reply:
I think we'll need this when we implement the terraform plan and apply operations.

Reply:
Why is root in the job package?  We can just leave that in the main package